src/main/java/org/snomed/snowstorm/core/data/services/ReferenceSetMemberService.java

```diff
@@ -60,6 +62,14 @@
 
 @Service
 public class ReferenceSetMemberService extends ComponentService {
+	private static final Function<ReferenceSetMember, Object[]> REFERENCE_SET_MEMBER_ID_SEARCH_AFTER_EXTRACTOR = referenceSetMember -> {
+		if (referenceSetMember == null) {
+			return null;
+		}
+
+		String id = referenceSetMember.getId();
+		return id == null ? null : SearchAfterHelper.convertToTokenAndBack(new Object[]{id});
+	};
```

Comment:
I referenced other areas in the code that used SearchAfterPage. From what I can tell, the function here is used to determine which record to scroll to. I've used the id of the member as that should be unique.